css/inner-page.css

```diff
@@ -434,6 +434,14 @@ ul#mysiderbar li {
     padding-left: 11px;
 }
 
+.mysidebar>li.mobile-sign-in {
+    padding: 0;
+}
+
+.mysidebar>li.mobile-sign-in a.loginText:hover {
+    background: white;
```

Comment:
We may not have a standard yet, but how about we work with hex here (`#fff`)?

Reply:
done